app/components/Instructions/Instructions.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function Instructions({ state, }) {
+  function buildInstructionArray(appState) {
+    const instructionList = [];
+    const {
+      pathStartingPoint,
+      pathEndingPoint,
+      startButtonPressed,
+      endButtonPressed,
+    } = appState;
+    const setGoBackText = buttonText => (
```

Comment:
get, not set. set is when you assign something. There's no assignment here, you call the function and GET back a string.

Reply:
done